Makefile

```diff
@@ -212,4 +212,8 @@ devdoc-clean:
 devdoc-update:
 	docker pull tendermint/devdoc
 
+proto-gen:
+	go mod vendor
```

Comment:
What's the rationale behind creating the `vendor/` directory?

Reply:
It's to be able to implement the `gogo.proto` file.